assets/pages/bundles-landing/components/bundlesGBStructureTest.jsx

```diff
@@ -0,0 +1,399 @@
+      props.currency.iso,
+      props.isoCountry,
+      props.intCmp,
+    );
+
+  const onClick = () => {
+    if (!props.contribError) {
+      window.location = contribAttrs.ctaLink;
+    }
+  };
+
+  return (
+    <Bundle {...contribAttrs}>
+      <p>
+        Your contribution funds and supports the&nbsp;Guardian&#39;s journalism
+      </p>
+      <ContribAmounts
+        onNumberInputKeyPress={onClick}
+        {...props}
+      />
+      <Secure />
+      <InlinePaymentLogos />
+
+      <CtaLink
+        ctaId={contribAttrs.ctaId.toLowerCase()}
+        text={contribAttrs.ctaText}
+        accessibilityHint={contribAttrs.ctaAccessibilityHint}
+        onClick={onClick}
+      />
+
+      {props.contribType === 'ONE_OFF' && !!contribAttrs.paypalCta &&
+      <PayPalContributionButton
+        buttonText={contribAttrs.paypalCta.text}
+        amount={Number(props.contribAmount.oneOff.value)}
+        referrerAcquisitionData={props.referrerAcquisitionData}
+        isoCountry={props.isoCountry}
+        // eslint-disable-next-line no-alert
+        errorHandler={e => alert(e)}
+        abParticipations={props.abTests}
+        additionalClass={props.contribError ? 'contrib-error' : ''}
+        canClick={!props.contribError}
+      />
+      }
+
+    </Bundle>
+  );
+
+}
+
+function SubscribeBundle(props: Object) {
+  return (
+    <Bundle {...props}>
+      <FeatureList listItems={bundles[props.subscriptionProduct].listItems} />
+      <CtaLink
+        ctaId={props.ctaId}
+        text={props.ctaText}
+        dataLinkName="bundlesLandingPageDigipackLink"
+        accessibilityHint={props.ctaAccessibilityHint}
+        url={props.ctaLink}
+      />
+    </Bundle>
+  );
+
+}
+
+
+// ----- Component ----- //
+
+function BundlesGBStructureTest(props: PropTypes) {
+
+  const subsLinks: SubsUrls = getSubsLinks(
+    props.intCmp,
+    props.campaign,
+    props.otherQueryParams,
+    props.abTests,
+    props.referrerAcquisitionData,
+  );
+  const digitalAttrs: SubscribeAttrs = getDigitalAttrs(subsLinks);
+  const paperAttrs: SubscribeAttrs = getPaperAttrs(subsLinks);
+  const paperDigitalAttrs: SubscribeAttrs = getPaperDigitalAttrs(subsLinks);
+
+  return (
+    <section className="bundles bundles--structure-test">
+      <div className="bundles__content gu-content-margin bundles__content--structure-test">
+        <div className="bundles__wrapper">
+          <h2 className="bundles__title">contribute</h2>
+          <ContributionBundle {...props} />
+          <h2 className="bundles__title">subscribe</h2>
+          <div className="bundles__divider" />
+          <SubscribeBundle {...digitalAttrs} subscriptionProduct="digital" />
+          <div className="bundles__divider" />
+          <SubscribeBundle {...paperAttrs} subscriptionProduct="paper" />
+          <div className="bundles__divider" />
+          <SubscribeBundle {...paperDigitalAttrs} subscriptionProduct="paperDigital" />
+        </div>
+      </div>
+      <CrossProduct />
```

Comment:
`CrossProduct` needs to go inside the div with `gu-content-margin` applied, otherwise this happens:

![cross-product](https://user-images.githubusercontent.com/5131341/33134051-95ec9764-cf96-11e7-92ec-7e1458728a33.png)

Reply:
👍